v21.2/data-domiciling-with-placement-restricted.md

```diff
@@ -0,0 +1,387 @@
+---
+title: Data Domiciling with ALTER DATABASE ... PLACEMENT RESTRICTED
+summary: Learn how to implement data domiciling using CockroachDB's multi-region SQL capabilities and the ALTER DATABASE ... PLACEMENT RESTRICTED statement.
+toc: true
+---
+
+{% include {{page.version.version}}/sql/data-domiciling-intro.md %}
+
+## Overview
+
+This page has instructions for data domiciling in [multi-region clusters](multiregion-overview.html) using the [`ALTER DATABASE ... PLACEMENT RESTRICTED`](placement-restricted.html) statement. At a high level, this process involves:
+
+1. Controlling the placement of specific row or table data using regional or global tables - specifically, the [`REGIONAL BY ROW`](multiregion-overview.html#regional-by-row-tables), [`REGIONAL BY TABLE`](multiregion-overview.html#regional-tables), and [`GLOBAL`](multiregion-overview.html#global-tables) table localities.
+1. Further restricting where the data in those regional tables is stored using the [`ALTER DATABASE ... PLACEMENT RESTRICTED`](placement-restricted.html) statement, which constrains the replicas for a row or table to be stored in only the [home regions](set-locality.html#crdb_region) associated with those rows or tables.
+
+For more information, see the sections below.
+
+## Prerequisites
+
+This page assumes you are already familiar with:
+
+- CockroachDB's [multi-region SQL abstractions](multiregion-overview.html).
+- The fact that CockroachDB stores your data in [a distributed key-value store, which is split into chunks called ranges](architecture/distribution-layer.html#overview).
+
+## Example
+
+In the following example, you will go through the process of configuring the [MovR](movr.html) data set using [multi-region SQL statements](multiregion-overview.html). Then, as part of implementing a data domiciling strategy, you will apply stricter-than-default replica placement settings using the [`ALTER DATABASE ... PLACEMENT RESTRICTED`](placement-restricted.html) statement. Finally, you will verify that the resulting replica placements are as expected using a combination of [replication reports](query-replication-reports.html), the [`SHOW RANGE FOR ROW`](show-range-for-row.html) statement, and queries against the [`crdb_internal.ranges_no_leases`](crdb-internal.html#ranges_no_leases) table.
+
+For the purposes of this example, the data domiciling requirement is to configure a multi-region deployment of the [MovR database](movr.html) such that data for EMEA-based users, vehicles, etc. is being stored on CockroachDB nodes running in EMEA localities.
+
+### Step 1. Start a simulated multi-region cluster
+
+{% include {{page.version.version}}/sql/start-a-multi-region-demo-cluster.md %}
```

Comment:
nit: `cockroach demo --global --nodes 9 --no-example-database --insecure` may be slow right now due to https://github.com/cockroachdb/cockroach/issues/76305, should we also include the `--multitenant=false` flag?